cmd/training-operator.v1/main.go

```diff
@@ -33,6 +33,7 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/healthz"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	"go.uber.org/zap/zapcore"
 
 	mpiv1 "github.com/kubeflow/training-operator/pkg/apis/mpi/v1"
 	mxnetv1 "github.com/kubeflow/training-operator/pkg/apis/mxnet/v1"
```

Comment:
The indentation for the import statements are not correct

Reply:
@terrytangyuan ok, `go fmt`